apollo-backend/src/main/resources/io/logz/apollo/dao/EnvironmentDao.xml

```diff
@@ -24,9 +24,15 @@
 
     <insert id="addEnvironment" useGeneratedKeys="true" keyProperty="id" keyColumn="id">
         INSERT INTO environment
-        (name, geo_region, availability, kubernetes_master, kubernetes_token, kubernetes_ca_cert, kubernetes_namespace, service_port_coefficient, require_deployment_message, require_health_check, additional_params)
+        (name, geo_region, availability, kubernetes_master, kubernetes_token, kubernetes_ca_cert, kubernetes_namespace, service_port_coefficient, require_deployment_message, require_health_check, concurrency_limit, additional_params)
```

Comment:
Made sure its not hurting the environment controller?

Reply:
Yap